smac/runhistory/runhistory.py

```diff
@@ -32,6 +32,10 @@ def __new__(
         return super().__new__(cls, config_id, instance_id, seed, budget)
 
 
+RunInfo = collections.namedtuple(
```

Comment:
seems to be repetitive compared to `RunKey`. Why do we need both?

Reply:
The difference is that this has the configuration object itself, while the `RunKey` has only the configuration ID. The idea is to pass all relevant information for execution to the target function, so that the target function can be executed in a different worker that does not have access to the runhistory.